src/components/App.css

```diff
@@ -0,0 +1,318 @@
+  width: 11.5rem;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  box-sizing: border-box;
+  border: none;
+  background-color: #0290ff;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  color: #fff;
+}
+
+.book-detail {
+  box-sizing: border-box;
+  width: 75rem;
+  height: 10.625rem;
+  margin-left: 6.25rem;
+  margin-top: 0.938rem;
+  display: flex;
+  justify-content: space-between;
+  padding: 2rem 7.438rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+}
+
+.book-chapter h4 {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 3.625rem 0 3.688rem;
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.79rem;
+  color: #121212;
+}
+
+.book-chapter p {
+  width: 13.25rem;
+  height: 1.313rem;
+  margin: 0.438rem 6.813rem 0.25rem 3.688rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.9rem;
+  letter-spacing: -0.4px;
+}
+
+.progress-bar h4 {
+  width: 2.938rem;
+  height: 2.438rem;
+  margin: 0.063rem 7.063rem 0 0.688rem;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  margin-top: 1rem;
+}
+
+.progress-bar p {
+  width: 5rem;
+  height: 1.125rem;
+  margin: 0.438rem 4.938rem 0 0.75rem;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+}
+
+.book-detail .desc h4 {
+  width: 7rem;
+  height: 1.125rem;
+  margin: 0 4rem 0 0;
+  opacity: 0.5;
+  font-family: 'Montserrat', sans-serif;
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.book-detail .desc h2 {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.progress-bar {
+  display: flex;
+  justify-content: space-between;
+  border-right: solid 1px #e8e8e8;
+}
+
+.book-detail .desc p {
+  width: 5.813rem;
+  height: 1.188rem;
+  margin: 0.25rem 0.875rem 0 0;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.868rem;
+  font-weight: 300;
+  color: #4386bf;
+}
+
+.book-detail .desc .links {
+  display: flex;
+  margin-top: 1.313rem;
+}
+
+.book-detail .desc .links p {
+  width: 4.563rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+}
+
+.book-detail .desc .links p button {
+  border: none;
+  background-color: white;
+  width: 3.375rem;
+  height: 1.188rem;
+  margin: -2px 0.938rem 0.125rem 0.875rem;
+  font-family: 'Roboto Slab', serif;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
```

Comment:
no repeating of the same rule. kindly group classes that have some common style to avoid repeated style and to make your code dry